www/css/state-machine.css

```diff
@@ -0,0 +1,115 @@
+.equation {
+	position: absolute;
+	border: 3px solid black;
+	padding: 4px 7px;
+	box-shadow: 2px 2px 19px #e0e0e0;
+	-o-box-shadow: 2px 2px 19px #e0e0e0;
+	-webkit-box-shadow: 2px 2px 19px #e0e0e0;
+	-moz-box-shadow: 2px 2px 19px #e0e0e0;
+	opacity:0.8;
+	filter:alpha(opacity=80);
+	cursor: move;
+	font-size: 11px;
+	text-align: center;
+	vertical-align: middle;
+	width: 100px;
+	height: auto;
+	max-height: 48px;
+	-webkit-transition:background-color 0.25s ease-in;
+	-moz-transition:background-color 0.25s ease-in;
+	transition:background-color 0.25s ease-in;
+}
+
+.quantity {
+	border: 3px solid;
+	position: absolute;
+	/*
+	* apparently giving high radius gives a capsule shape
+	* http://stackoverflow.com/questions/18794947
+	* this seems to be a hack leaving it just in case it is needed
+	*/
+	border-radius: 500px;
+	/*border-radius: 10%/50%; this is height dependent*/
+	padding: 4px 7px;
+	text-align: center;
+	width: 100px;
+	height: auto;
+	max-height: 48px;
+	box-shadow: 2px 2px 19px #e0e0e0;
+	-o-box-shadow: 2px 2px 19px #e0e0e0;
+	-webkit-box-shadow: 2px 2px 19px #e0e0e0;
+	-moz-box-shadow: 2px 2px 19px #e0e0e0;
+	opacity:0.8;
+	filter:alpha(opacity=80);
+	cursor: move;
+	font-size: 11px;
+	text-align: center;
+	vertical-align: middle;
+	-webkit-transition:background-color 0.25s ease-in;
+	-moz-transition:background-color 0.25s ease-in;
+	transition:background-color 0.25s ease-in;
+}
+
+.equationWrapper, .quantityWrapper {
+	overflow: hidden!important;
+	-ms-text-overflow:ellipsis;
+	text-overflow:ellipsis;
+	width: 100%;
+	height: auto;
+	display: -webkit-box;
+	display: -moz-box;
+	display: -o-box;
+	-webkit-box-orient: vertical;
+	-moz-box-orient: vertical;
+	-o-box-orient: vertical;
+	-webkit-line-clamp: 4;
+	-moz-line-clamp: 4;
+	-o-line-clamp: 4;
+	line-height: 11px;        /* fallback */
+	max-height: 44px;
+}
+
+.circle{
+	width: 60px;
+	height: 60px;
+	border: 3px solid black;
+	border-radius: 50%;
+	background-color: grey;
+	color: white;
+	text-align: center;
+	vertical-align: middle;
+}
+
+/*.circleWrapper{
+	padding: 10px;
+}*/
+
+/* css for path color and end point color for node structure for model UI*/
+path {
+	cursor:pointer;
+	pointer-events: visiblestroke;
+	stroke:rgb(92, 150, 188);
+	stroke-width: 2px;
+}
+
+.ep {
```

Comment:
This is useful in jsplumb library, that is what the jsplumb jquery integration demo used as well. ".ep"